(CHore ) #2059 PHASE 1: Feed: Update welcome popup text

yaroslav-fedyshyn-nordwhale opened the pull request:
# Description

Change the text of the welcome feed card for phase1

About #2059 

# How Has This Been Tested?

See updated welcome feed card.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

## Screenshot:
<img width="472" alt="1" src="https://user-images.githubusercontent.com/49862004/85835179-4266c000-b79d-11ea-9a40-021a311c79ce.png">

LiavGut commented:
@yaroslav-fedyshyn-nordwhale The Button label's font-weight is not as per design.

yaroslav-fedyshyn-nordwhale commented:
> @yaroslav-fedyshyn-nordwhale The Button label's font-weight is not as per design.

I fixed the fontWeight and updated the screen in the description

yaroslav-fedyshyn-nordwhale commented:
> @yaroslav-fedyshyn-nordwhale
> The Character spacing seems too big, is it like all the other buttons?

@LiavGut yes, it is like the other buttons.

LiavGut commented:
@yaroslav-fedyshyn-nordwhale did you changed it after my last comment? I don't see the new design.

yaroslav-fedyshyn-nordwhale commented:
@LiavGut See new letter spacing - 
<img width="469" alt="1" src="https://user-images.githubusercontent.com/49862004/86144384-488ed080-bafe-11ea-9194-852e7b3080d8.png">
